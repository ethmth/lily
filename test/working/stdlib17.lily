
let a::float = [-6.43, 1.25, 2112.2]
a.selectionsort()

print(a)