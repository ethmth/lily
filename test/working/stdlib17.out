[-6.430000, 1.250000, 2112.200000]
